test/Graphing/Calls/Spec.hs

```diff
@@ -10,15 +10,18 @@ import Algebra.Graph
 import           "semantic" Data.Graph (Graph (..), topologicalSort)
 import           Data.Graph.ControlFlowVertex
 import qualified Data.Language as Language
+import qualified Data.Map as Map
+import           Data.Maybe (fromJust)
 import           Semantic.Graph
 import qualified System.Path as Path
 
 callGraphPythonProject :: Path.RelFile -> IO (Semantic.Graph.Graph ControlFlowVertex)
 callGraphPythonProject path = runTaskOrDie $ do
   let proxy = Proxy @'Language.Python
-  let lang = Language.Python
+      lang = Language.Python
+  SomeParser parser <- pure . fromJust $! parserForLanguage analysisParsers Language.Python
```

Comment:
Does `Just (SomeParser parser)` work? Will give better diagnostics if this ever happens to be Maybe.

Reply:
Unfortunately not, because it requires a `MonadFail` instance.